### ISSUE TITLE
Virtually no copper nodes in Dun Morogh

### ISSUE THREAD

**Issue by Hr46ph** [dalaranwow/dalaran-wow#4564]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
In the entire area of Dun Morogh there are virtually no copper mining nodes.

**Expected behaviour**: Tell us what should happen instead
Find copper mining nodes to mine when running around.

**Steps to reproduce the problem**:

1. Run around Dun Morogh with "Find Minerals" enabled and mining skill learned. You'll find a few.

Dun Morogh is supposed to have the most mining nodes of all starting areas.

**Include proofs for this behaviour**
![coppernodes](https://cloud.githubusercontent.com/assets/6275310/23963685/658152b8-09b2-11e7-9198-2869a6ba2ddd.JPG)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
the maximum pooling limit in that zone is 15 - it should be around 80 like in the barrens

**Comment by Rushor** [comment]
DB/Gameobject: Increase limit of "Master Mineral Pool - Dun Morogh" to 88

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4564
* commit: c3a32eec5262b331e0e397944e1a59be47c9b025

**Comment by Rushor** [comment]
DB/Gameobject: Increase limit of "Master Mineral Pool - Dun Morogh" to 88

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4564
* commit: c3a32eec5262b331e0e397944e1a59be47c9b025

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
